qiita_db/metadata_template/util.py

```diff
@@ -337,6 +331,36 @@ def get_invalid_sample_names(sample_names):
     return inv
 
 
+def get_invalid_column_names(column_names):
+    """Get a list of column names that are not SQL compliant
+
+    Parameters
+    ----------
+    column_names : iterable
+        Iterable containing the column names to check.
+
+    Returns
+    -------
+    list
+        List of str objects where each object is an invalid column name.
+
+    References
+    ----------
+    .. [1] postgresql SQL-SYNTAX-IDENTIFIERS: https://goo.gl/EF0cUV.
+    """
+    valid_initial_char = letters
+    valid_rest = set(letters+digits+'_')
```

Comment:
Excellent point but it doesn't: https://travis-ci.org/biocore/qiita/builds/133477520